src/test/java/com/bitmovin/api/examples/CreateHLSEncodingWithAES128.java

```diff
@@ -0,0 +1,239 @@
+        List<H264Representation> h264Representations = new ArrayList<>();
+        h264Representations.add ( new H264Representation(null, 240, null, 400L, ProfileH264.HIGH));
+        h264Representations.add(new H264Representation(null, 360, null, 800L, ProfileH264.HIGH));
+        h264Representations.add(new H264Representation(null, 480, null, 1200L, ProfileH264.HIGH));
+        h264Representations.add(new H264Representation(null, 720, null, 2400L, ProfileH264.HIGH));
+        h264Representations.add(new H264Representation(null, 1080, null, 4800L, ProfileH264.HIGH));
+
+        // Encoding Configuration - Audio
+        AACAudioConfig aacConfiguration = new AACAudioConfig();
+        aacConfiguration.setBitrate(128000L);
+        aacConfiguration.setRate(48000f);
+        aacConfiguration = bitmovinApi.configuration.audioAAC.create(aacConfiguration);
+
+        for (H264Representation representation : h264Representations) {
+            H264VideoConfiguration videoConfiguration = new H264VideoConfiguration();
+            videoConfiguration.setHeight(representation.getHeight());
+            videoConfiguration.setBitrate(representation.getBitrate()*1000);
+            videoConfiguration.setProfile(ProfileH264.HIGH);
+            videoConfiguration = bitmovinApi.configuration.videoH264.create(videoConfiguration);
+            representation.setConfiguration(videoConfiguration);
+
+            // Encoding Configuration - Streams
+            InputStream inputStreamAudio = new InputStream();
+            inputStreamAudio.setInputPath(HTTPS_INPUT_PATH);
+            inputStreamAudio.setInputId(input.getId());
+            inputStreamAudio.setSelectionMode(StreamSelectionMode.AUTO);
+            inputStreamAudio.setPosition(0);
+
+            InputStream inputStreamVideo = new InputStream();
+            inputStreamVideo.setInputPath(HTTPS_INPUT_PATH);
+            inputStreamVideo.setInputId(input.getId());
+            inputStreamVideo.setSelectionMode(StreamSelectionMode.AUTO);
+            inputStreamVideo.setPosition(0);
+
+            Stream audioStream = new Stream();
+            audioStream.setCodecConfigId(aacConfiguration.getId());
+            audioStream.setInputStreams(Collections.singleton(inputStreamAudio));
+            audioStream = bitmovinApi.encoding.stream.addStream(encoding, audioStream);
+            representation.setStream(audioStream);
+
+            Stream videoStream = new Stream();
+            videoStream.setCodecConfigId(videoConfiguration.getId());
+            videoStream.setInputStreams(Collections.singleton(inputStreamVideo));
+            videoStream = bitmovinApi.encoding.stream.addStream(encoding, videoStream);
+            representation.setStream(videoStream);
+
+            // AES - Configuration
+            AesEncryptionDrm aesEncryptionDrm = new AesEncryptionDrm();
+            aesEncryptionDrm.setKey("<SET_YOUR_STRING_KEY>"); // 128 bit (16 bytes)  ex.: cab5b529ae28d5cc5e3e7bc3fd4a544d
```

Comment:
Maybe let's have this info as a const (err I mean static) at the top of the file, but it's a minor quibble.